lib/Espfc/src/Device/InputESPNOW.h

```diff
@@ -0,0 +1,187 @@
+#ifndef _INPUT_DEVICE_INPUT_ESPNOW_H_
+#define _INPUT_DEVICE_INPUT_ESPNOW_H_
+
+#include "esp_wifi.h"
+#include <WiFi.h>
+#include <esp_now.h>
```

Comment:
First of all, if we use this library https://github.com/yoursunny/WifiEspNow then all targets will be covered out of the box, including esp8266.